jpo-ode-plugins/src/main/java/us/dot/its/jpo/ode/plugin/j2735/builders/BrakeSystemStatusBuilder.java

```diff
@@ -29,8 +31,16 @@ private BrakeSystemStatusBuilder() {
    public static J2735BrakeSystemStatus genericBrakeSystemStatus(JsonNode brakesStatus) {
       J2735BrakeSystemStatus genericBrakesStatus = new J2735BrakeSystemStatus();
 
-      genericBrakesStatus.setWheelBrakes(BitStringBuilder.genericBitString(brakesStatus.get("wheelBrakes"),
+      JsonNode wheelBrakes = brakesStatus.get("wheelBrakes");
+      if (wheelBrakes.asText().trim().equals("00000")) {
+         ObjectNode wheelBrakesUnavailable = JsonUtils.newNode();
+         wheelBrakesUnavailable.put("wheelBrakes", "10000");
```

Comment:
Is this "10000" value translated to "unavailable"? Can we add a comment to that affect so we know why we are setting it?

Reply:
Addressed